TheThingsNetwork/lorawan-stack
# Update Japanese translations

#### Summary

This PR updates the Japanese translations and closes translation gaps.

Enterprise counterpart: https://github.com/TheThingsIndustries/lorawan-stack/pull/3874

#### Changes



- Add updated `ja.json` locale file
- Add `ja` as a supported locale by default in the translation script, so that new messages get added automatically to the file

#### Testing

1. Go to the Console
2. Set the language to Japanese
3. Observe all messages translated and no English remaining

#### Notes for Reviewers

I've revisited the translation script as well. We can now create the CSVs for the translations like so:
`node tools/mage/translations.js --export-for-intepreters`

This will create a CSV that contains all strings of all supported non-default locales (currently Japanese only) that are not yet translated. This file can then be sent to a translator to fill out and the be ingested back into the locale file.

#### Checklist
